x-pack/plugins/observability_solution/apm/server/routes/services/get_throughput.ts

```diff
@@ -67,7 +72,9 @@ export async function getThroughput({
             ...environmentQuery(environment),
             ...kqlQuery(kuery),
             ...termQuery(TRANSACTION_NAME, transactionName),
+            ...(parsedFilters.filter ?? []),
           ],
+          must_not: [...(parsedFilters.must_not ?? [])],
```

Comment:
Same for the `parsedFilters.must_not`?
```suggestion
          must_not: parsedFilters.must_not,
```